openssl/src/x509/mod.rs

```diff
@@ -827,6 +827,33 @@ impl X509NameBuilder {
         }
     }
 
+    /// Add a field entry by str with a specific type. (ex: 19 for V_ASN1_PRINTABLESTRING)
+    ///
+    /// This corresponds to [`X509_NAME_add_entry_by_txt`].
+    ///
+    /// [`X509_NAME_add_entry_by_txt`]: https://www.openssl.org/docs/man1.1.0/crypto/X509_NAME_add_entry_by_txt.html
+    pub fn append_entry_by_text_with_type(
+        &mut self,
+        field: &str,
+        value: &str,
+        ty: i32,
```

Comment:
This argument should be some kind of real type rather than a raw number.

Reply:
Just to make sure I understand: you would like an enum type defined?

Reply:
Not exactly an enum type, but something like https://docs.rs/openssl/0.10.30/src/openssl/ssl/mod.rs.html#595

Reply:
Thanks for the example, I'll get a revision in either tonight or tomorrow.